presets/shared.yml

```diff
@@ -211,6 +211,7 @@ maxConnectionAttempts: 15
 baseRetryDelay: 750
 connectionPoolSize: 10
 maxSubscriptions: 300
+restDeploymentTool: symbol-bootstrap
```

Comment:
Why does ``restDeploymentTool`` needs to be update-able?  Can there be any other deployment tool?

Reply:
It's just the extension point just in case somebody for some reason wants to customize the string, same with the version and the date

Reply:
Ok.  How can you tell if someone is using bootstrap when this is change?
If you want an extension point then I would think the bootstrap should always be present.  maybe like ``Bootstrap:<extension data>``

Reply:
If the user really wants to mess up with the rest.json, he can manually update it after being generated. I'm just trying to not leave any place hardcoded

Reply:
(There are other rest.json properties to be moved to shared, I would do at some point)